undefined reference to `SSL_CIPHER_find'

Hello,

I'm struggling with building the newest zcoin 0.13.5.6. I'm getting an error:

 CXXLD    zcoind
tor/src/common/libor-crypto.a(tortls.o): In function `tor_tls_classify_client_ciphers':
tortls.c:(.text+0x4f7): undefined reference to `SSL_CIPHER_find'

Is there any fix for this?

Idk, after updating the repo once more and rebuilding again, now I get error:

src/or/libtor.a(main.o): In function `tor_main':
main.c:(.text+0x7f61): undefined reference to `evutil_secure_rng_set_urandom_device_file'
src/common/libor-event.a(compat_libevent.o): In function `tor_gettimeofday_cache_clear':
compat_libevent.c:(.text+0x840): undefined reference to `event_base_update_cache_time'
collect2: error: ld returned 1 exit status

What's that about?

Reopening this as I have encountered the issue myself.
I think there is some issue with the version of Tor we have.
The issue only seems to occur when building the dependancies for Linux in the `depends` folder

steps to reproduce:
```
cd depends
make HOST=x86_64-linux-gnu -j`nproc` (or x86_64-unknown-linux-gnu)
cd ..
./autogen.sh
./configure --prefix=`pwd`/depends/x86_64-linux-gnu
sudo make -j`nproc`
```

Output:
```
src/or/libtor-testing.a(src_or_libtor_testing_a-main.o): In function `tor_run_main':
main.c:(.text+0x934d): undefined reference to `evutil_secure_rng_set_urandom_device_file'
collect2: error: ld returned 1 exit status
Makefile:4785: recipe for target 'src/test/fuzz/fuzz-consensus' failed
make[3]: *** [src/test/fuzz/fuzz-consensus] Error 1
make[3]: *** Waiting for unfinished jobs....
src/or/libtor-testing.a(src_or_libtor_testing_a-main.o): In function `tor_run_main':
main.c:(.text+0x934d): undefined reference to `evutil_secure_rng_set_urandom_device_file'
collect2: error: ld returned 1 exit status
Makefile:4805: recipe for target 'src/test/fuzz/fuzz-diff' failed
make[3]: *** [src/test/fuzz/fuzz-diff] Error 1
src/or/libtor-testing.a(src_or_libtor_testing_a-main.o): In function `tor_run_main':
main.c:(.text+0x934d): undefined reference to `evutil_secure_rng_set_urandom_device_file'
collect2: error: ld returned 1 exit status
Makefile:4795: recipe for target 'src/test/fuzz/fuzz-descriptor' failed
make[3]: *** [src/test/fuzz/fuzz-descriptor] Error 1
make[3]: Leaving directory '/drive/zcoin-personal/zcoin/src/tor'
Makefile:3472: recipe for target 'all' failed
make[2]: *** [all] Error 2
make[2]: Leaving directory '/drive/zcoin-personal/zcoin/src/tor'
Makefile:11219: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/drive/zcoin-personal/zcoin/src'
Makefile:690: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -1159,7 +1159,7 @@ if test x$need_bundled_univalue = xyes; then
 fi
 
 AX_SUBDIRS_CONFIGURE([src/secp256k1], [[--disable-shared], [--with-pic], [--with-bignum=no], [--enable-module-recovery], [--enable-experimental], [--enable-module-ecdh]])
-AX_SUBDIRS_CONFIGURE([src/tor], [[--disable-unittests], [--disable-system-torrc], [--disable-systemd], [--disable-lzma], [--disable-zstd], [--disable-asciidoc], [--with-openssl-dir=$($PKG_CONFIG --variable=libdir openssl)], [CFLAGS=$($PKG_CONFIG --cflags openssl) -O2]])
+AX_SUBDIRS_CONFIGURE([src/tor], [[--disable-unittests], [--disable-system-torrc], [--disable-systemd], [--disable-lzma], [--disable-zstd], [--disable-asciidoc]])
 
 
 AC_OUTPUT

